cmd/registry/scoring/score_test.go

```diff
@@ -373,6 +374,37 @@ func TestProcessScoreFormulaError(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc: "unsupported artifact type",
+			setup: func(ctx context.Context, client connection.Client, adminClient connection.AdminClient) {
+				deleteProject(ctx, adminClient, t, "score-formula-test")
```

Comment:
These are using the wrong `t` variable, and will cause problems if failures occur. See #565 for more info. If you plan to make a separate PR that fixes this in all of our tests, then feel free to leave it unaddressed in this PR.

Reply:
Thanks for catching this! I will fix this separately across the codebase. There are instances of this in controller and scoring tests.